Sources/TuistGenerator/Generator/ConfigGenerator.swift

```diff
@@ -76,11 +76,11 @@ final class ConfigGenerator: ConfigGenerating {
         let buildConfigurations = Set(projectBuildConfigurations).union(targetBuildConfigurations)
         let configurationsTuples: [(BuildConfiguration, Configuration?)] = buildConfigurations
```

Comment:
This code was already there but I find `BuildConfiguration` and `Configuration` a bit ambiguous. What do they represent? If renaming them is a huge change, we can perhaps name the elements of the tuple to make it easier to understand.

Reply:
Renaming them should be fine, but not sure how I should name them; maybe `Configuration`->`BuildConfigurationSettings`?

Reply:
Do you feel like the changed naming is a good solution or a different wording would be better @pepibumur ? It is the last hurdle before merging this 🙂